x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/readme.tsx

```diff
@@ -56,15 +56,11 @@ export function Readme({
       ) : (
         <EuiText>
           {/* simulates a long page of text loading */}
-          <p>
-            <EuiSkeletonText lines={5} />
-          </p>
-          <p>
-            <EuiSkeletonText lines={6} />
-          </p>
-          <p>
-            <EuiSkeletonText lines={4} />
-          </p>
+          <EuiSkeletonText lines={5} />
```

Comment:
Updated that component as it was triggering a invalid dom warning when running jest tests making debugging those tests harder.